packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -15,9 +15,9 @@ export const oidcModelInstancePayloadGuard = z
 export type OidcModelInstancePayload = z.infer<typeof oidcModelInstancePayloadGuard>;
 
 export const oidcClientMetadataGuard = z.object({
-  applicationType: z.enum(['web', 'native']),
   redirectUris: z.string().array(),
   postLogoutRedirectUris: z.string().array(),
+  logoUri: z.string().optional(),
```

Comment:
nit: [zod string](https://github.com/colinhacks/zod#strings) has a built-in URL constraint. should we take advantage of this?